applications/asset_tracker_v2/src/modules/atv2_led_module.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <logging/log.h>
+#include "atv2_led_module.h"
+#include <device.h>
+#include <caf/events/led_event.h>
+#include "events/led_state_event.h"
+#include <caf/events/module_state_event.h>
+
+#if (IS_ENABLED(CONFIG_LED_USE_PWM))
```

Comment:
The config name is wrong. Note that it has nothing to do with PWM - you select board version and this information is actually provided.